indico/modules/events/editing/client/js/editing/timeline/TimelineItem.jsx

```diff
@@ -80,15 +81,23 @@ export default function TimelineItem({block, index}) {
                 <time dateTime={serializeDate(createdDt, moment.HTML5_FMT.DATETIME_LOCAL_SECONDS)}>
                   {serializeDate(createdDt, 'LLL')}
                 </time>{' '}
+                {block.modifiedDt && (
+                  <span
+                    className="review-comment-edited"
+                    title={Translate.string('On {modificationDate}', {
```

Comment:
Do we really need a translatable string for this? I'd just put the date/time in the tooltip...

Also, using another `<time>` element like above would probably be better in terms of semantics

Reply:
fine by me. this is copied straight from the `CommentItem` btw, I will also change it there then

Reply:
BTW you might also want to change `const {createdDt, isUndone} = block;` above to destructure the modifiedDt. it's a bit strange to get one directly from the block and the other via destructuring